formatter.rb

```diff
@@ -0,0 +1,33 @@
+RSpec::Support.require_rspec_core 'formatters/base_text_formatter'
+
+class SimpleFormatter < RSpec::Core::Formatters::BaseTextFormatter
+  RSpec::Core::Formatters.register self,
+    :example_passed, :example_pending, :example_failed, :dump_pending, :dump_failures, :dump_summary
+
+  def example_passed(message)
+    # Do nothing
+  end
+
+  def example_pending(message)
+    # Do nothing
+  end
+
+  def example_failed(message)
+    # Do nothing
+  end
+
+  def dump_pending(message)
+    # Do nothing
+  end
+
+  def dump_failures(message)
+  end
+
+  def dump_summary(message)
+    colorizer = ::RSpec::Core::Formatters::ConsoleCodes
+
+    output.puts "\nFinished in #{message.formatted_duration} " \
+       "(files took #{message.formatted_load_time} to load)\n" \
+       "#{message.colorized_totals_line(colorizer)}\n"
+  end
+end
```

Comment:
Thanks to [patkoperwas](http://stackoverflow.com/users/3295284/patkoperwas) for he made it possible just print the number of pass/failed examples with colors. [Here's](http://stackoverflow.com/questions/37553229/print-only-the-number-of-failed-examples-with-rspec?noredirect=1#comment62596136_37553229) his answer that helped me.